.gitignore

```diff
@@ -0,0 +1,230 @@
+# The MIT License (MIT)
```

Comment:
I realise this was generated by Visual Studio and it seems like a good choice for the `.gitignore` file.

However, another option you could consider is the one provided by GitHub at <https://github.com/github/gitignore/blob/master/VisualStudio.gitignore>. This has the advantage of being community maintained, so that it stays better up to date. For instance, I notice you manually included `project.lock.json` below but this would be unnecessary with the GitHub version as it is already present.

Reply:
The github version uses "Creative Commons Zero v1.0 Universal", but Coimbra is MIT.

Reply:
I'm not sure that's an issue. Based on the `README.md` at <https://github.com/github/gitignore>, it seems that these `.gitignore` files are used when populating new projects, but this step was unfortunately missed when creating this repo. Most repos won't have the "Creative Commons Zero v1.0 Universal" licence and yet applying the CC0 `.gitignore` seems to be fine in these cases.

My understanding is that `CC0` was selected as it entails no copyright claim, so the files can be used anywhere. However, I am admittedly not clear on the specifics, and I would have felt more confident if the licence were explicitly called out within the file.

I would have preferred if this were created through the tooling for initialising the file in the repo management options, so that we don't have to explicitly commit it, which would give us greater certainty that we were doing the right thing here.

Using the Visual Studio version could well be the better option here to ensure we don't violate any licencing terms.